walker.go

```diff
@@ -78,8 +79,8 @@ type Walker struct {
 	walk   *fspb.Walk
 	walkMu sync.Mutex
 
-	// Outpath, if non-empty, is where Walk will be written to.
-	Outpath string
+	// Function to use to write the Walk.
+	WriteWalk WriteCallback
```

Comment:
If you make the change above renaming the callback, I'd rename the field to `RunCallback` as well.

Reply:
Done